### PR TITLE
Issues Fixed

### PR DESCRIPTION
**Pull request by cq-mohtashim-ali**

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="" vcs="" />
+  </component>
+</project>
```
Comment (MasoodAhmad) [inline .idea/vcs.xml:6]: put .idea in git ignore

File: src/sample/Main.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+import java.io.IOException;
+import java.util.Base64;
+import java.util.Timer;
+
+public class Main extends Application {
+
+    static String text = "A";
+    static BufferedImage screenFullImage;
+    static String image;
+    private static int inactivityTime = 60;
```
Comment (cq-ubaid-khan) [inline src/sample/Main.java:23]: put in config

File: src/sample/MouseLog.java [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+                    count = 0;
+                    temp = 0;
+                    SIGNAL = true;
+                    if (KeyLog.SIGNAL == true && KeyLog.SIGNAL2 == false){
+                        String name =  Controller.name;
+                        Message message = new Message(name,"Disconnected");
+                        String objects = message.toJson();
+                    handler.sendObject(objects);
+                    handler.cancel();
+                    SIGNAL2 = true;
```
Comment (cq-ubaid-khan) [inline src/sample/MouseLog.java:40]: no proper indentation

File: src/sample/WebsocketClientEndpoint.java [hunk 4]
```diff
@@ -1,60 +1,88 @@
...
         });
     }
+
     public void connect() throws DeploymentException, IOException, URISyntaxException {
         WebSocketContainer container = ContainerProvider.getWebSocketContainer();
         ClientEndpointConfig config = ClientEndpointConfig.Builder.create()
                 .configurator(new Configurator())
                 .build();
-        container.connectToServer(this, config, new URI(url));
+        container.connectToServer(this, config, new URI("ws://localhost:8080/echo"));
```
Comment (cq-ubaid-khan) [inline src/sample/WebsocketClientEndpoint.java:42]: why is not in config?

File: .idea/vcs.xml (hunk 1 above)
Comment (cq-mohtashim-ali) [reply .idea/vcs.xml:6]: OK

**Pull request merged**